src/main/java/net/imglib2/loops/LoopBuilder.java

```diff
@@ -183,7 +224,18 @@ private boolean cursorIsFast( IterableInterval<?> image )
 	 */
 	public LoopBuilder< T > multiThreaded()
 	{
-		this.multiThreaded = Objects.requireNonNull( MultiThreadSetting.MULTI );
+		return multiThreaded( Parallelization.getTaskExecutor() );
```

Comment:
Please add clarification to the javadoc that this will use multi-threading if run in a parallelized context.
(If I understand correctly, if run using `Parallelization.runSingleThreaded( ... )` then `multiThreaded()` does nothing?)

Reply:
done